src/app/components/core/configuration-manager/configuration.module.ts

```diff
@@ -27,10 +26,9 @@ const routes: Routes = [
     FormsModule,
     CommonModule,
     RouterModule.forChild(routes),
-    NgProgressModule,
     SharedModule,
     DirectivesModule,
-    TreeModule
+    TreeModule.forRoot()
```

Comment:
`forRoot()` method returns an NgModule and its provider dependencies. This is required by the latest version(8.1.0) of `angular-tree-component`. Otherwise it will not work.